README.md

```diff
@@ -229,13 +231,18 @@ return if the new bird **wasn't** created successfully?
 
 ## Conclusion
 
-We've now seen how to handle the `create` action, and take data from the
-**body** of a request using the `params` hash. When Rails receives a request
-where the `Content-Type` is set to `application/json`, it will automatically
-load the request body into the `params` hash.
+We have now learned how to handle the `create` action. In the next lesson, we'll
+explore the `params` hash further, and talk about ways to refactor our code
+using additional features of the `params` hash.
 
-In the next lesson, we'll explore the `params` hash further, and talk about ways
-to refactor our code using additional features of the `params` hash.
+## Check For Understanding
+
+Before you move on, make sure you can answer the following questions:
+
+1. When making a `POST` request as opposed to a `GET` request, what additional
+   property needs to be passed along with the `method` and `headers`?
```

Comment:
To clarify:

When using `fetch` to make a `POST` request as opposed to a `GET` request, what additional property needs to be passed along with the `method` and `headers`?